### PR TITLE
initial commit of an ase template

### PR DESCRIPTION
**Pull request by yossidahan**
Initial commit of a template to create an AppService Environment with a Service Plan and a  Logic App

### CODE REVIEW

File: appserviceenvironment-serviceplan-logicapp/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,270 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "defaultValue": "West Europe",
```
Comment (singhkay) [inline appserviceenvironment-serviceplan-logicapp/azuredeploy.json]: Remove default location

File: appserviceenvironment-serviceplan-logicapp/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,262 @@
+﻿{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "location": {
+            "type": "string",
+            "metadata": {
```
Comment (singhkay) [inline appserviceenvironment-serviceplan-logicapp/azuredeploy.json]: Needs allowed values for the regions Core RPs exist in today. See https://github.com/Azure/azure-quickstart-templates/blob/master/101-create-availability-set/azuredeploy.json

File: appserviceenvironment-serviceplan-logicapp/metadata.json [hunk 3]
```diff
@@ -0,0 +1,7 @@
+﻿{
+  "itemDisplayName": "AppServiceEnvironment  (ASE) with an AppService Plan",
```
Comment (singhkay) [inline appserviceenvironment-serviceplan-logicapp/metadata.json]: Nit: Shouldn't this be App Service Environment?

**Comment by singhkay** [comment]
Can you change the name of the folder to app-service-environment-serviceplan-logicapp for easier readability?

File: appserviceenvironment-serviceplan-logicapp/azuredeploy.json [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+            "defaultValue": 1,
+            "metadata": {
+                "description": "The number of IP Addresses to create for the environment"
+            }
+        },
+        "worker1Size": {
+            "type": "string",
+            "defaultValue": "small",
+            "allowedValues": [
+                "small",
```
Comment (singhkay) [inline appserviceenvironment-serviceplan-logicapp/azuredeploy.json]: Do these smal, medium, large values correspond to Standard_A1, A2, A3 VM sizes?

File: appserviceenvironment-serviceplan-logicapp/readme.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+Built by: [YOSSIDAHAN](https://github.com/yossidahan)
+
+This template creates a new AppService Environment on a specified Subnet in a pre-existign Virtual Netowrk. 
+It proceeds to create an AppService hosting plan preparing the environment to recieve Web, API and Logic apps.
+Finally - it deploys a sample Logic app that calls http://www.example.com on a schedule
+
+The template assumes a 'Classic' network has been pre-provisioned that includes a subnet onto which to deploy the environment.
```
Comment (singhkay) [inline appserviceenvironment-serviceplan-logicapp/readme.md]: What is the 'Classic' network?

Comment (yossidahan) [inline appserviceenvironment-serviceplan-logicapp/readme.md]: I've hopefully clarified the comment in the readme.md file - I believe that ASE requires a V1 network

File: appserviceenvironment-serviceplan-logicapp/azuredeploy.json (hunk 4 above)
Comment (yossidahan) [inline appserviceenvironment-serviceplan-logicapp/azuredeploy.json]: I believe so

**Pull request closed without merging**